pkg/distributor/ha_tracker_http.go

```diff
@@ -27,12 +27,13 @@ type haTrackerStatusPageContents struct {
 }
 
 type haTrackerReplica struct {
-	UserID       string        `json:"userID"`
-	Cluster      string        `json:"cluster"`
-	Replica      string        `json:"replica"`
-	ElectedAt    time.Time     `json:"electedAt"`
-	UpdateTime   time.Duration `json:"updateDuration"`
-	FailoverTime time.Duration `json:"failoverDuration"`
+	UserID           string        `json:"userID"`
+	Cluster          string        `json:"cluster"`
+	Replica          string        `json:"replica"`
+	LastElectionTime time.Time     `json:"lastElectionTime"`
+	ElectedAt        time.Time     `json:"electedAt"`
```

Comment:
Worth renaming the field here to match the field used elsewhere, `electedLastSeenTime` ?